org.sasview.sasview.yml

```diff
@@ -0,0 +1,58 @@
+id: org.sasview.sasview
+runtime: org.kde.Sdk
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 6.7
+command: sasview
+finish-args:
+  - "--socket=fallback-x11"
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--device=dri"
+  - "--persist=."
+modules:
+  - name: hdf5
+    buildsystem: simple
+    build-commands:
+      - ./hdf5-1.14.5/configure --prefix=/app
+      - make -j4
+      - make install
+    sources:
+      - type: archive
+        url: https://support.hdfgroup.org/releases/hdf5/v1_14/v1_14_5/downloads/hdf5-1.14.5.tar.gz
+        sha256: ec2e13c52e60f9a01491bb3158cb3778c985697131fc6a342262d32a26e58e44
+  - name: qhull
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DBUILD_STATIC_LIBS=OFF"
+    sources:
+      - type: archive
+        url: https://github.com/qhull/qhull/archive/refs/tags/2020.2.tar.gz
+        sha256: 59356b229b768e6e2b09a701448bfa222c37b797a84f87f864f97462d8dbc7c5
+    cleanup:
+      - "/bin"
+      - "/share/man"
+      - "/share/doc"
+  - shared-modules/glu/glu-9.json
+  - python3-requirements.json
+  - name: sasview
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} "." --no-build-isolation
+    post-install:
+      - mkdir -p /app/share/metainfo/
+      - cp build_tools/application_metadata/org.sasview.sasview.metainfo.xml /app/share/metainfo/
+      - mkdir -p /app/share/icons/hicolor/512x512/apps/
+      - cp build_tools/application_metadata/org.sasview.sasview.png /app/share/icons/hicolor/512x512/apps/org.sasview.sasview.png
+      - mkdir -p /app/share/applications/
+      - cp build_tools/application_metadata/org.sasview.sasview.desktop /app/share/applications/
```

Comment:
```suggestion
      - install -Dm0644 build_tools/application_metadata/org.sasview.sasview.metainfo.xml -t /app/share/metainfo/
      - install -Dm0644 build_tools/application_metadata/org.sasview.sasview.png -t /app/share/icons/hicolor/512x512/apps/org.sasview.sasview.png
      - install -Dm0644 build_tools/application_metadata/org.sasview.sasview.desktop -t /app/share/applications/
```

Reply:
Thanks for the suggestion. This does look a lot cleaner.